back-nest/src/sсrapers/scrapers.module.ts

```diff
@@ -3,10 +3,10 @@ import { Module } from '@nestjs/common';
 import { ScraperTelemartService } from './scraper-telemart.service';
 import { ScraperController } from './scrapers.controller';
 import { ScraperRozetkaService } from './scraper-rozetka.service';
-import { PrismaService } from '../../prisma/prisma.service';
+import { ProductService } from '../products/products.services';
 
 @Module({
   controllers: [ScraperController],
-  providers: [ScraperTelemartService, ScraperRozetkaService, PrismaService],
+  providers: [ScraperTelemartService, ScraperRozetkaService, ProductService],
```

Comment:
Remove Products service. It's global and you don't need to add it into providers!